interview-form.html

```diff
@@ -0,0 +1,173 @@
+        <!--why the weird span? cuz CSS can't be fucking serious http://forums.mozillazine.org/viewtopic.php?f=25&t=291007&start=0&st=0&sk=t&sd=a-->
+        <span class="text-input">
+          <input placeholder="Enter your full name" type="text" name="name" id="name-input">
+        </span>
+        <div class="error-text">Your name is required</div>
+      </div>
+
+      <div id="email" class="form-input-group">
+        <label for="email-input">Email</label>
+        <span class="text-input">
+          <input placeholder="Enter your email" type="text" name="email" id="email-input">
+        </span>
+        <div class="error-text">Email is not valid</div>
+      </div>
+
+      <div id="github" class="form-input-group">
+        <label for="github-input">Github</label>
+        <span class="text-input">
+          <input placeholder="Enter your Github User URL" type="text" name="github" id="github-input">
+        </span>
+        <div class="error-text">The github user doesn't exist</div>
+      </div>
+
+      <div id="stream" class="form-input-group">
+        <label for="stream-input">Stream have in 12th</label>
+        <span class="text-input">
+          <input placeholder="Enter your Stream,ex.PCM+CS" type="text" name="stream" id="stream-input">
+        </span>
+        <div class="error-text">This can't be empty</div>
+      </div>
+      
+      <hr class="separator">
+
+      <div id="languages" class="form-input-group">
+        <label for="languages-input">Languages</label>
+        <div class="option-group">
+          <input type="checkbox" name="languages" id="languages-swift" value="swift">
+          <label for="languages-swift">Swift</label>
+          <input type="checkbox" name="languages" id="languages-typescript" value="typescript">
+          <label for="languages-typescript">Typescript</label>
+          <input type="checkbox" name="languages" id="languages-kotlin" value="kotlin">
+          <label for="languages-kotlin">Kotlin</label>
+          <input type="checkbox" name="languages" id="languages-golang" value="golang">
+          <label for="languages-golang">Go</label>
+          <input type="checkbox" name="languages" id="languages-c#" value="c#">
+          <label for="languages-c#">C#</label>
+          <input type="checkbox" name="languages" id="languages-javascript" value="javascript">
+          <label for="languages-javascript">JavaScript</label>
+          <input type="checkbox" name="languages" id="languages-java" value="java">
+          <label for="languages-java">Java</label>
+          <input type="checkbox" name="languages" id="languages-python" value="pyhton">
+          <label for="languages-python">Python</label>
+        </div>
+        <div class="error-text">Choose at least one language</div>
+      </div>
+
+
+      <div id="frameworks" class="form-input-group">
+        <label for="frameworks-input">Frameworks</label>
+        <div class="option-group">

```

Comment:
Since there are way too many frameworks available and all of them solve similar problems, it makes sense to add another option of "others".